.github/ISSUE_TEMPLATE/testplan.md

```diff
@@ -20,6 +20,7 @@ as well as an upgrade of the previous version of Teleport.
 - [ ] Labels
   - [ ] Static Labels
   - [ ] Dynamic Labels
+  - [ ] Resource-based Labels
```

Comment:
What are resource based labels? Is it the labels we get from the compute provider the instance is running in? Like AWS EC2 tags and Azure VM labels?

Reply:
They're labels that can be created at run time with a `server_info` resource: https://goteleport.com/docs/admin-guides/management/admin/labels/#apply-resource-based-labels

Reply:
Thank you 🙏 
I've used `server_info` but was not aware of the "resource based labels"

Reply:
We can retain the name, but can we please make this element a link to that document? Seeing "server_info labels", I also wouldn't know what it is.

Reply:
I added the docs link and also mentioned that it uses `server_info` inline :+1: